frontend/app/services/SubscriptionService.scala

```diff
@@ -123,4 +125,14 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String], val
   def createSubscription(memberId: MemberId, joinData: JoinForm, customerOpt: Option[Stripe.Customer]): Future[SubscribeResult] = {
     zuora.request(Subscribe(memberId, customerOpt, tierPlanRateIds(joinData.plan), joinData.name, joinData.deliveryAddress))
   }
+
+  def getPaymentSummary(memberId: MemberId): Future[PaymentSummary] = {
+    for {
+      subscription <- getSubscriptionStatus(memberId)
+      invoiceItems <- zuora.query[InvoiceItem](s"SubscriptionId='${subscription.current}'")
+    } yield {
+      val sortedInvoiceItems = sortInvoiceItems(invoiceItems)
```

Comment:
I made a change: https://github.com/guardian/membership-frontend/commit/8fa26222b3d8b6c26329697950520879ce6dcff7

Is this okay?